spanner/test/cloudexecutor/executor/internal/inputstream/handler.go

```diff
@@ -142,6 +209,48 @@ func (h *CloudStreamHandler) startHandlingRequest(ctx context.Context, req *exec
 	}
 }
 
+// verifyCloudTraceExportedTraces fetches the traces exported from client application using
+// Cloud Trace API to cross verify if end to end tracing is working or not.
+func (h *CloudStreamHandler) verifyCloudTraceExportedTraces(ctx context.Context, traceId string) error {
```

Comment:
looks like this method is getting called only once after all the actions complete. If that is right, and TraceClient is getting used only at this place then there is no need to create it way before and store it in the execution flow context. You can create the trace client here and close it once the verification completes. 
WDYT?
Is the traceclient used somewhere else?

Reply:
No, trace client is only used at the end and doesn't need to be created at the start. thanks for pointing out.